apps/system/test/unit/ftu_launcher_test.js

```diff
@@ -140,6 +140,20 @@ suite('launch ftu >', function() {
   suite('stepReady', function() {
     setup(function() {
       FtuLauncher._stepsList = [];
+      FtuLauncher._done = false;
+      FtuLauncher._skipped = false;
```

Comment:
Saddens me a bit to have more internal setup here but I guess that's what we get for having a non-instantiable `FtuLauncher` :)